### ISSUE TITLE
addStateInfoToParentActive` is marked as required

### ISSUE THREAD

**Issue by martino87r** [ui-router/react#180]
I've recently started to rewrite a frontend and bumped all versions of libraries, with latest releases i get a strange warning:

```
Warning: Failed prop type: The prop `addStateInfoToParentActive` is marked as required in `Sref`, but its value is `undefined`.
    in Sref
    in UISref (created by SideMenu)
```

I'm using material-ui v1 stable (released a few days ago), and latest ui-router-react (and redux)

Below code is used for the ui:

```
// Main navigation feature of the app
import React, { Component } from 'react'
import { UISref } from '@uirouter/react'

import AppBar from '@material-ui/core/AppBar'
import Drawer from '@material-ui/core/Drawer'
import Toolbar from '@material-ui/core/Toolbar'
import Typography from '@material-ui/core/Typography'
import Button from '@material-ui/core/Button'
import IconButton from '@material-ui/core/IconButton'
import List from '@material-ui/core/List'
import ListItem from '@material-ui/core/ListItem';
import ListItemIcon from '@material-ui/core/ListItemIcon';
import ListItemText from '@material-ui/core/ListItemText';
import Divider from '@material-ui/core/Divider'
import Grid from '@material-ui/core/Grid';

import MenuIcon from '@material-ui/icons/Menu'
import HomeIcon from '@material-ui/icons/Home'
import DeviceHubIcon from '@material-ui/icons/DeviceHub'

import { withStyles } from '@material-ui/core/styles';


class SideMenu extends Component {

  constructor(props) {
    super(props)

    this.state = {
      open: false
    }

    this.toggleMenu = this.toggleMenu.bind(this)
  }

  toggleMenu(open) {
    return () => {
      this.setState({ open })
    }
  }

  render() {
    return (
      <div>
        <IconButton onClick={this.toggleMenu(true)} color="secondary" aria-label="Menu">
          <MenuIcon />
        </IconButton>
        <Drawer open={this.state.open} onClose={this.toggleMenu(false)} >
          <div
            tabIndex={0}
            role="button"
            onClick={this.toggleMenu(false)}
            onKeyDown={this.toggleMenu(false)}>
            <List>
              <UISref to="app.index">
                <ListItem button>
                  <ListItemIcon>
                    <HomeIcon />
                  </ListItemIcon>
                  <ListItemText primary="Home" />
                </ListItem>
              </UISref>
            </List>
          </div>
        </Drawer>
      </div>
    )
  }
}

const styles = {
  root: {
    maxHeight: 50,
  },
  toolbar: {
    minHeight: 50
  }
}

class Nav extends Component {

  constructor(props) {
    super(props)
    this.state = {
      openMenu: false
    }
  }

  render() {
    return (
      <AppBar position="static" className={this.props.classes.root}>
        <Toolbar disableGutters={true} className={this.props.classes.toolbar}>
          <Grid container alignItems="center" spacing={0}>
            <Grid item>
              <SideMenu />
            </Grid>
            <Grid item>
              <Typography variant="title" color="inherit">
                MCNet 3 - RR2 DEV
              </Typography>
            </Grid>
            <Grid item>
            </Grid>
          </Grid>
        </Toolbar>
      </AppBar>
    )
  }
}

export default withStyles(styles)(Nav)
```

**Comment by elboman** [comment]
Thanks for filing the issue. This problem has already been fixed (c7aa299). When I updated to the new Context API I forgot to remove the `isRequired` in the props.

But we didn't release a new patch version.

You can ignore it for now, I'll release a new version asap and it should go away.

**Comment by elboman** [comment]
@martino87r I just released `0.8.0`, should be fixed now!

**Comment by martino87r** [comment]
Thanks for the promptly answer and kudos on your work, ui-router rocks!
